### PR TITLE
Fav track fix

### PR DESCRIPTION
**Pull request by mahimshridhar**
My first pull request in this repo, suggestions are welcome.

### CODE REVIEW

**Comment by SjefVelthuizen** [comment]
unsubscribe



Op 1 september 2019 3:47 uur schreef Mahim M S <notifications@github.com>:



My first pull request in this repo, suggestions are welcome.

You can view, comment on, or merge this pull request online at:

  https://github.com/nukeop/nuclear/pull/464

Commit Summary
fixed favorite track duplication issue
changed variable name
File Changes
M app/components/TrackRow/index.js (13)
Patch Links:
https://github.com/nukeop/nuclear/pull/464.patch
https://github.com/nukeop/nuclear/pull/464.diff

File: app/components/TrackRow/index.js [hunk 1]
```diff
@@ -51,6 +51,12 @@ class TrackRow extends React.Component {
     );
   }
   
+  addToFavorites() {
```
Comment (nukeop) [inline app/components/TrackRow/index.js]: The only suggestion I have is to rename this function to something like `canAddToFavorites`, to indicate the intent.

**Comment by nukeop** [comment]
Very nice, thank you for the contribution. I only have one suggestion and then we'll be ready to merge.

File: app/components/TrackRow/index.js (hunk 1 above)
Comment (mahimshridhar) [reply app/components/TrackRow/index.js]: Thank you for the suggestion, i have changed the function name to be more specific.

**Comment by nukeop** [comment]
Seems to work well, nice job.

**Pull request merged**